example/flake.nix

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
Are we keeping `default.nix` and `flake.nix` for the example?

Both work, but people might look at this and think they need both.

Reply:
I thought I'd removed it, whoops.